fapolicy_analyzer/tests/test_object_list.py

```diff
@@ -71,24 +71,30 @@ def strip_markup(markup):
 def test_status_markup(widget):
     view = widget.get_object("treeView")
 
+    st_trust = '<span color=\"green\"><b><u>ST</u></b></span>'
+    at_trust = '<span color=\"green\"><b><u>AT</u></b></span>'
+    st_untrust = '<span color=\"red\"><b>ST</b></span>'
+    at_untrust = '<span color=\"red\"><b>AT</b></span>'
+    untrust = '<span color="green"><b><u>U</u></b></span>'
```

Comment:
If you are using single quotes on the outside, you don't need to escape the double quotes on the inside. There are a few places where this needs changed.